code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -64,9 +64,12 @@
 	name = "roller bed"
 	icon = 'icons/obj/rollerbed.dmi'
 	icon_state = "down"
+	var/up_state = "up"
+	var/down_state = "down"
```

Comment:
Formatting nitpick: These should go down with `var/folded`, So we have all the new subtype variables together:
```dm
/obj/structure/bed/roller
	name = "roller bed"
	//etc 
	var/up_state
	var/down_state
	var/folded
```
Also autodoc would be nice for the vars.